auction-server/src/auction/repository/models.rs

```diff
@@ -648,8 +660,18 @@ pub trait Database<T: ChainTrait>: Debug + Send + Sync + 'static {
 
 #[async_trait]
 impl<T: ChainTrait> Database<T> for DB {
+    #[instrument(
+        target = "metrics",
+        fields(
+            category = "db_queries",
+            result = "success",
+            name = "add_auction",
```

Comment:
maybe we should name this something else, to distinguish it from the instrument name. This `name` field influences the name of the metrics, but the instrument/span name is what determines the naming of the traces.